minesweeper_model/str_input.py

```diff
@@ -0,0 +1,28 @@
+field=[]
```

Comment:
Can I please ask you to put all of this code into a function? Maybe call it "str_input_to_mine_coords"